### ISSUE TITLE
Console loads duplicate trace spans

### ISSUE THREAD

**Issue by andresuribe87** [TBD54566975/ftl#3260]
Go to the traces view
Wait for 20ish seconds, go to other tabs.
The result is reloading the flame graph, which causes duplicate traces to appear

**Comment by deniseli** [comment]
pretty sure it's a simple js bug in the rendering template. I'll grab this next if wes doesn't beat me to it

**Comment by wesbillman** [comment]
I think this is being caused by these errors

```
index-7hTRnJxG.js:280 useSchema - streamSchema - Connect error: ConnectError: [unknown] missing EndStreamResponse
    at Zt.from (index-7hTRnJxG.js:279:9628)
    at index-7hTRnJxG.js:280:41319Caused by: missing EndStreamResponse
index-7hTRnJxG.js:444 Console service - streamEvents - Connect error: ConnectError: [unknown] missing EndStreamResponse
    at Zt.from (index-7hTRnJxG.js:279:9628)
    at index-7hTRnJxG.js:280:41319Caused by: missing EndStreamResponse
```

I would guess that these lead to us re-requesting the events starting from the beginning after this happens. I wonder what is causing these errors though 🤔

**Comment by wesbillman** [comment]
The duplicate entries should be fixed with #3266
